crawler/rojak_pantau/spiders/pilkada_2018_kompascom.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+from dateparser import parse
+from datetime import datetime
+from scrapy import Request
+from scrapy.loader import ItemLoader
+from scrapy.exceptions import CloseSpider
+
+from rojak_pantau.items import News
+from rojak_pantau.i18n import _
+from rojak_pantau.util.wib_to_utc import wib_to_utc
+from rojak_pantau.spiders.base import BaseSpider
+
+class Pilkada2018KompascomSpider(BaseSpider):
+
+    def __init__(self, media_id, election_id):
```

Comment:
You need to call init function in the base class. otherwise, some informations needed by the base class will not be available.

Reply:
oh ya miss that